target/linux/x86/config-6.6

```diff
@@ -197,10 +198,13 @@ CONFIG_INSTRUCTION_DECODER=y
 # CONFIG_INTEL_LDMA is not set
 # CONFIG_INTEL_PCH_THERMAL is not set
 # CONFIG_INTEL_POWERCLAMP is not set
+CONFIG_INTEL_RAPL=y
+CONFIG_INTEL_RAPL_CORE=y
 # CONFIG_INTEL_SCU_PCI is not set
 # CONFIG_INTEL_TCC_COOLING is not set
 # CONFIG_INTEL_VSEC is not set
-# CONFIG_IOSF_MBI is not set
+CONFIG_IOSF_MBI=y
+# CONFIG_IOSF_MBI_DEBUG is not set
```

Comment:
This is activated in all subtarget configurations:
```
$ git grep IOSF_MBI 
target/linux/x86/64/config-6.6:CONFIG_IOSF_MBI=y
target/linux/x86/64/config-6.6:# CONFIG_IOSF_MBI_DEBUG is not set
target/linux/x86/config-6.6:# CONFIG_IOSF_MBI is not set
target/linux/x86/generic/config-6.6:CONFIG_IOSF_MBI=y
target/linux/x86/generic/config-6.6:# CONFIG_IOSF_MBI_DEBUG is not set
target/linux/x86/geode/config-6.6:CONFIG_IOSF_MBI=y
target/linux/x86/geode/config-6.6:# CONFIG_IOSF_MBI_DEBUG is not set
target/linux/x86/legacy/config-6.6:CONFIG_IOSF_MBI=y
target/linux/x86/legacy/config-6.6:# CONFIG_IOSF_MBI_DEBUG is not set
```
Please remove all subtaregt config options.